server/odc-service/src/main/java/com/oceanbase/odc/service/db/schema/DBSchemaIndexService.java

```diff
@@ -153,7 +157,7 @@ public QueryDBObjectResp listDatabaseObjects(@NonNull @Valid QueryDBObjectParams
             resp.setDatabases(databases.stream().filter(e -> e.getName().toLowerCase().contains(key))
                     .collect(Collectors.toList()));
         }
-        Pageable pageable = PageRequest.of(0, 1000, Sort.by(Direction.DESC, DBColumnEntity_.DATABASE_ID));
+        Pageable pageable = PageRequest.of(0, MAX_SEARCH_SIZE, Sort.by(Direction.DESC, DBColumnEntity_.DATABASE_ID));
```

Comment:
1000 --> 10000? Is this change expected?

Reply:
I modified it from 10000 to 5000, which means that we will query at most 5000 records. It is excepted and has been tested. The time cost is about 200ms.